client/src/Viewer.tsx

```diff
@@ -165,6 +168,57 @@ const ViewerCitations = () => {
   )[0]?.polygons;
 
   if (!polygons) return null;
+  const rectsForUnion: [number, number][][][] = polygons.map((poly) => {
+    const x1 = poly[0];
+    const y1 = poly[1];
+    const x2 = poly[4];
+    const y2 = poly[5];
+    return [
+      [
+        [x1, y1],
+        [x2, y1],
+        [x2, y2],
+        [x1, y2],
+      ],
+    ];
+  });
+  const unioned = polygonClipping.union(rectsForUnion);
+  const ringToPath = (ring: number[][]) => {
+    return (
+      ring
+        .map(
+          (coords, i) =>
+            `${i === 0 ? "M" : "L"} ${coords[0] * multiple},${
+              coords[1] * multiple
+            }`
+        )
+        .join(" ") + " Z"
+    );
+  };
+  const allPaths: string[] = [];
+  unioned.forEach((polygon) => {
+    polygon.forEach((ring) => {
+      allPaths.push(ringToPath(ring));
+    });
+  });
+
+  const highlightSvg = (
+    <svg
+      style={{
```

Comment:
I prefer to put all the static styles into a CSS class and only use `style` for the dynamic styles.

Reply:
Updated.